javascriptv3/example_code/cross-services/transcription-app/src/index.ts

```diff
@@ -0,0 +1,283 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+ABOUT THIS NODE.JS EXAMPLE: This example works with AWS SDK for JavaScript version 3 (v3),
+which is available at https://github.com/aws/aws-sdk-js-v3. This example is in the 'AWS SDK for JavaScript v3 Developer Guide' at
+https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/transcribe-app.html.
+
+Purpose:
+index.ts is part of a tutorial demonstrating how to build and deploy an app that transcribes and displays
+voice recordings for authenticated users. To run the full tutorial, see
+https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/transcribe-app.html.
+
+Inputs (replace in code):
+- COGNITO_ID
+- ID_TOKEN
+- BUCKET
+- REGION
+- IDENTITY_POOL_ID
+
+Running the code:
+For more information, see https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/transcribe-app.html.
+
+[Outputs | Returns]:
+
+*/
+// snippet-start:[transcribe.JavaScript.recording-app.config]
+// Import the required AWS SDK clients and commands for Node.js
+const AWS = require("aws-sdk");
+require("./helper.ts");
+require("./recorder.ts");
+const { IAMClient, GetUserCommand } = require("@aws-sdk/client-iam");
+const { CognitoIdentityClient } = require("@aws-sdk/client-cognito-identity");
+const {
+  fromCognitoIdentityPool,
+} = require("@aws-sdk/credential-provider-cognito-identity");
+const { S3RequestPresigner } = require("@aws-sdk/s3-request-presigner");
+const { createRequest } = require("@aws-sdk/util-create-request");
+const { formatUrl } = require("@aws-sdk/util-format-url");
+const {
+  TranscribeClient,
+  StartTranscriptionJobCommand,
+  GetTranscriptionJobCommand,
+} = require("@aws-sdk/client-transcribe");
+const {
+  S3,
+  S3Client,
+  PutObjectCommand,
+  GetObjectCommand,
+  CreateBucketCommand,
+  HeadBucketCommand,
+  ListObjectsCommand,
+  DeleteObjectCommand,
+} = require("@aws-sdk/client-s3");
+const { path } = require("path");
+const fetch = require("node-fetch");
+let COGNITO_ID = "cognito-idp.eu-west-1.amazonaws.com/USER_POOL_ID";
+const idToken = getToken();
+const loginData = {
+  [COGNITO_ID]: idToken,
+};
+
+// Create the parameters
+const params = {
+  Bucket: "BUCKET_NAME", // Amazon S3 Bucket to store transcriptions
```

Comment:
"Amazon S3" -> "Amazon Simple Storage Service (Amazon S3)" on first mention in a code file. Also "Bucket" -> "bucket" (not a proper name)